src/reactviews/media/slickgrid.css

```diff
@@ -111,6 +111,9 @@
 
 .slick-cell .grid-cell-value-container.missing-value {
     font-style: italic;
+    display: block;
+    background-color: var(--vscode-editor-inactiveSelectionBackground);
+    color: var(--vscode-editor-selectionForeground);
```

Comment:
Also, it looks like most styles are applied in [`style()` from table.ts](https://github.com/microsoft/vscode-mssql/blob/d8ed8efe5ed29a6e02b5c882f9882dc7cdd22cea/src/reactviews/pages/QueryResult/table/table.ts#L552) with the colors themselves defined [here](https://github.com/microsoft/vscode-mssql/blob/d8ed8efe5ed29a6e02b5c882f9882dc7cdd22cea/src/reactviews/pages/QueryResult/table/interfaces.ts#L71)